core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.directive.html

```diff
@@ -18,6 +17,7 @@
   side-navigation-bar .oppia-sidebar-menu::after {
     background: rgba(0,0,0,0.2);
     content: '';
+    display: none;
```

Comment:
Why is this file being changed at first place? 

Second, why are we touching the css of this file?

Reply:
I'm sorry, I messed up something while rolling back side-navigation-bar I think, I'll fix it

Reply:
Ah, OK. Please do take care of such things. In general, take a pass over your changes when you push the PR in files changed section to make sure all the changes look good to you :).